zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -500,6 +531,7 @@ ZEXTERN int ZEXPORT z_deflatePending OF((z_streamp strm,
         return deflatePending(strm, pending, bits);
     FINISH_WITH_ERR("deflatePending is not supported!");
```

Comment:
Indeed zlib does something like this:
`strm->msg = (char *)"incorrect length check";`

from zlib.h:
In the error case, msg may be set but then points to a static string (which must not be deallocated).

What should do zlibWrapper in case of error? Which error code should be returned? The list:
```
#define Z_OK            0
#define Z_STREAM_END    1
#define Z_NEED_DICT     2
#define Z_ERRNO        (-1)
#define Z_STREAM_ERROR (-2)
#define Z_DATA_ERROR   (-3)
#define Z_MEM_ERROR    (-4)
#define Z_BUF_ERROR    (-5)
#define Z_VERSION_ERROR (-6)
```